src/Ormolu/Printer/Meat/Pragma.hs

```diff
@@ -28,10 +53,19 @@ p_pragma :: PragmaTy -> String -> R ()
 p_pragma ty c = do
   txt "{-# "
   txt $ case ty of
-    Language -> "LANGUAGE"
+    Language _ -> "LANGUAGE"
     OptionsGHC -> "OPTIONS_GHC"
     OptionsHaddock -> "OPTIONS_HADDOCK"
   space
   txt (T.pack c)
   txt " #-}"
   newline
+
+-- | Classify a 'LanguagePragma'.
+classifyLanguagePragma :: String -> LanguagePragmaClass
+classifyLanguagePragma = \case
+  "ImplicitPrelude" -> ImplicitPrelude
+  "CUSKs" -> CUSKs
+  str
+    | "No" `isPrefixOf` str -> Disabling
```

Comment:
`NondecreasingIndentation` also starts with "No", you might have to break the string into case-separated chunks first.